src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public interface Figure {
+    String getColor();
+
+    double getArea();
+
+    void draw();
```

Comment:
The methods `draw()` and `getArea()` are not conceptually close to each other. Consider splitting this interface into smaller ones with a single responsibility. For example, you could have one interface for drawing and another for calculating the area.